packages/next/client/future/image.tsx

```diff
@@ -373,136 +369,151 @@ export default function Image({
   }
 
   const [blurComplete, setBlurComplete] = useState(false)
+  let widthInt = getInt(width)
+  let heightInt = getInt(height)
+  const qualityInt = getInt(quality)
 
   if (process.env.NODE_ENV !== 'production') {
     if (!src) {
-      throw new Error(
+      // Since react doesn't show the stack trace and we don't have a `src` to help identify the image with the error,
+      // we instead render an image with the error message embedded into the image
+      console.warn(
         `Image is missing required "src" property. Make sure you pass "src" in props to the \`next/image\` component. Received: ${JSON.stringify(
-          { width, height, quality }
+          { width, height, quality, className, style }
         )}`
```

Comment:
Should we also log the `img` ref so they can highlight the image in their devtools?

Reply:
Good point! Fixed in [cffee12](https://github.com/vercel/next.js/pull/38847/commits/cffee12faa7e28c03e02870e595c69da6d110a88)